packages/gatsby-theme-api-docs/src/components/info-value.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import {
+  Markdown,
+  useISO310NumberFormatter,
+  designSystem,
+} from '@commercetools-docs/ui-kit';
+
+const InfoValue = (props) => {
+  const value = props.children;
+  const valueType = props.type || typeof value;
+  const formatNumber = useISO310NumberFormatter();
+
+  const customCodeStyle = css`
+    font-size: ${designSystem.typography.fontSizes.small};
+  `;
+
+  console.log(valueType, value);
+
+  switch (valueType) {
+    case 'boolean':
+      return props.isQueryParameter ? (
+        <>
+          :{' '}
```

Comment:
can't we just move this out of the component? it seems not very reusable as it's assuming a certain formatting is out of the component